src/point.cpp

```diff
@@ -37,6 +37,16 @@ point point::rotate( int turns, const point &dim ) const
     return *this;
 }
 
+float point::distance( const point &rhs ) const
+{
+    return std::sqrt( static_cast<float>( std::pow( x - rhs.x, 2 ) + std::pow( y - rhs.y, 2 ) ) );
```

Comment:
Looks like you need to `#include <cmath>` in `point.cpp`.

Reply:
Thanks! Wanted to fix the above LGTM complains here. Compiled locally, but does seemingly not work on every system.